.cirrus.yml

```diff
@@ -681,11 +681,15 @@ podman_machine_task:
     # Required_pr_labels does not apply to non-PRs.
     # Do not run on tags, branches, [CI:BUILD], or [CI:DOCS].
     only_if: *not_tag_branch_build_docs
+    only_if: *is_pr
```

Comment:
This will simply override the previous entry.  Oh, but on examining the `not_tag_branch_build_docs` alias, it's already bypassing PRs:

```yaml
    only_if: &not_tag_branch_build_docs >-
        $CIRRUS_PR != '' &&
        $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' &&
        $CIRRUS_CHANGE_TITLE !=~ '.*CI:BUILD.*'
```

unless I'm reading that wrongly.

Reply:
ya.  I think all you need to do is remove 684 and 721

Reply:
done